index.js

```diff
@@ -210,6 +210,8 @@ function isSourceElement(opts, node) {
     case "TypeAliasDeclaration": // Typescript
     case "ExportAssignment": // Typescript
     case "ExportDeclaration": // Typescript
+    case "OperationDefinition": // GraphQL
+    case "FragmentDefinition": // GraphQL
```

Comment:
I think there's only these 2, according to https://facebook.github.io/graphql/#sec-Appendix-Grammar-Summary.Query-Document

Maybe I'm misinterpreting you, though?